app/cdap/components/Cloud/Profiles/AutoScaleBadge/index.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import T from 'i18n-react';
+import styled from 'styled-components';
+import Tooltip from '@material-ui/core/Tooltip';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+const PREFIX = 'features.Cloud.Profiles';
+
+const Badge = styled.span`
+  color: #fff;
+  font-size: 0.85rem;
+  background: var(--brand-primary-color);
+  padding: 2px 6px;
+  display: inline-block;
+  border-radius: 20px;
+  margin-left: 6px;
+  cursor: pointer;
+`;
+
+const BadgeTooltip = withStyles(() => {
```

Comment:
nit: I think you could avoid using withStyles here and use something like:

```javascript
const StyledTooltip = styled(Tooltip)`
  background-color: #999;
  font-size: 1rem;
`;
```

Reply:
styled() adds css directly to the Tooltip, but here we pass styles as a prop.